service/controller/app/resource/chartoperator/create.go

```diff
@@ -132,6 +132,16 @@ func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			}
 
 			r.logger.Debugf(ctx, "deleted release %#q", releaseName)
+		case helmclient.StatusDeployed:
+			r.logger.Debugf(ctx, "release %#q deployed", releaseName)
+			r.logger.Debugf(ctx, "triggering charts reconciliation")
+
+			err = r.triggerReconciliation(ctx, cr)
```

Comment:
Could you add a comment here for why we need this?

The comments in the method are good but I think having something here too would be useful.